cfgrammar/src/lib/header.rs

```diff
@@ -139,16 +154,17 @@ impl<'input> GrmtoolsSectionParser<'input> {
         Self { src, required }
     }
 
-    pub fn parse(&'_ self) -> Result<(Vec<ValBind<'_>>, usize), HeaderError> {
+    #[allow(clippy::type_complexity)]
+    pub fn parse(&'_ self) -> Result<(HashMap<(&'_ str, Span), Value<'_>>, usize), HeaderError> {
```

Comment:
Oops, we really *don't* want this `Span` in the key part of the hash table.

Reply:
Should be fixed in [79fb45b](https://github.com/softdevteam/grmtools/pull/544/commits/79fb45b811682e274553cad36b6993069944e50f)